apps/frontend/src/pages/Build/BuildDetail.tsx

```diff
@@ -317,13 +324,19 @@ const CompareScreenshot = ({ diff }: { diff: Diff }) => {
           }
         >
           <img
-            className={clsx("absolute", visible && "opacity-disabled")}
+            className={clsx(
+              "absolute",
+              visible && "opacity-disabled",
+              // Firefox fixes
+              contained && "max-h-full",
+              "left-0 right-0 mx-auto",
```

Comment:
I don't think it works if the comparison screenshot is smaller than the diff overlay, but maybe we have the bug today.